moirae/estimators/online/utils/model.py

```diff
@@ -32,6 +33,14 @@ class BaseCellWrapper(ModelWrapper):
     """
     Base link between the :class:`~moirae.model.base.CellModel` and the numpy-only interface of
     the filter implementations.
+
+    Args:
+        cell_model: Model which defines the physics of the system being modeled

```

Comment:
Should we include the degradation model wrapper as a part of every CellModel wrapper? I could imagine wanting to see the effect of degradation even if I'm not adjusting its parameters.

Reply:
The `CellModelWrapper` is not allowed to touch the A-SOH, so I don't think the `DegrdationModel` should be included there. I don't think I understood the reason why we should include it here...